### PR TITLE
Update README.md, fix spelling.

### PR DESCRIPTION
**Pull request by Noschvie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,16 +129,16 @@ There are also some text replacements available which will be replaced by detail
 
 
 ### Static Monitors
-In addition to reading Monitors from Docker labels, AutoKuma can create Monitors from files, this can be usefull if you have want AutoKuma to manage monitors which aren't directly related to a container. 
+In addition to read Monitors from Docker labels, AutoKuma can create Monitors from files. This can be usefull if you have want AutoKuma to manage monitors which aren't directly related to a container.
```
Comment (BigBoot) [inline README.md:132]: I think reading is actually correct here.

File: README.md [hunk 2]
```diff
@@ -178,7 +178,7 @@ Kuma CLI is a Command-Line Interface (CLI) tool for managing and interacting wit
 
 ## 🔧 How to Install
 
-Binaries for windows linux and mac are provided for [GitHub Releases](https://github.com/BigBoot/AutoKuma/releases/latest), additionally Kuma CLI can be installed using `cargo`:
+Binaries for Windows Linux and Mac are provided for [GitHub Releases](https://github.com/BigBoot/AutoKuma/releases/latest) and additionally Kuma CLI can be installed using `cargo`:
```
Comment (BigBoot) [inline README.md:181]: I guess you could add the missing `,` after Windows while you're at it :smile:

File: README.md (hunk 1 above)
Comment (Noschvie) [reply README.md:132]: "to read" is correct.

Comment (BigBoot) [reply README.md:132]: ChatGPT seems to agree with me:

The correct sentence is:

“In addition to reading Monitors from Docker labels, AutoKuma can create Monitors from files.”

Here’s why: In this context, “reading” is a gerund, which is a verb that functions as a noun in the sentence. The phrase “In addition to” is a preposition that is typically followed by a noun or a gerund (verb-ing). Therefore, “reading” is the correct form to use after “In addition to”. The verb “read” in the second sentence is in the base form, which is not grammatically correct in this context. So, the first sentence is the correct one.

**Pull request closed without merging**